exporters/otlp/internal/config.go

```diff
@@ -0,0 +1,37 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package retry provides request retry functionality that can perform
+// configurable exponential backoff for transient errors and honor any
+// explicit throttle responses received.
+package internal // import "go.opentelemetry.io/otel/exporters/otlp/internal"
+
+import (
+	"fmt"
+	"path"
+	"strings"
+)
+
+// CleanPath returns cleaned URL path. Replace with default path if path is nil
+func CleanPath(URLPath string, defaultPath string) string {
+	tmp := strings.TrimSpace(URLPath)
+	if tmp == "" {
+		return defaultPath
+	}
+	tmp = path.Clean(tmp)
```

Comment:
The clean operation can result in an empty string (i.e. `/dir/..`) which is returned as `"."`.

```suggestion
	tmp := path.Clean(strings.TrimSpace(URLPath))
	if tmp == "." {
		return defaultPath
	}
```